mLRS/CommonTx/mavlink_interface_tx.h

```diff
@@ -397,12 +409,48 @@ void tTxMavlink::handle_msg_serial_out(fmav_message_t* msg)
     if (msg->sysid != vehicle_sysid || msg->compid != MAV_COMP_ID_AUTOPILOT1) return;
 
     switch (msg->msgid) {
-    case FASTMAVLINK_MSG_ID_EXTENDED_SYS_STATE:{
+    case FASTMAVLINK_MSG_ID_EXTENDED_SYS_STATE: {
         fmav_extended_sys_state_t payload;
         fmav_msg_extended_sys_state_decode(&payload, msg);
         vehicle_is_flying = (payload.landed_state == MAV_LANDED_STATE_IN_AIR) ? 1 : 0;
-        }break;
+        } break;
+    }
+
+    // MissonPlanner style link quality, for autopilot only
+    if (msg_seq_initialized) {
+        msg_received_cnt++;
```

Comment:
yes
we could change that but on the other hand this metric is just for temporary use

like now it counts how many "gaps" we have, if taking the difgference of the seq numbers one counts how many mavlink packets are lost, so the current count will be "better", but it might be better comparable to the LQ.

pl discuss with @jlpoltrack what you prefer, I don't care :)